lib/features/identity_creator/presentation/identity_creator_controller.dart

```diff
@@ -701,4 +769,31 @@ class IdentityCreatorController extends BaseController {
       }
     });
   }
+
+  void onLocalFileDropZoneListener({
+    required BuildContext context,
+    required DropDoneDetails details,
+    required double maxWidth
+  }) async {
+    FocusManager.instance.primaryFocus?.unfocus();
+    final listFileInfo = await onDragDone(context: context, details: details);
+    final listImages = listFileInfo
+      .where((fileInfo) => fileInfo.isInline == true)
+      .toList();
+
+    if (listImages.isEmpty && listFileInfo.isNotEmpty && context.mounted) {
+      appToast.showToastErrorMessage(
+        context,
+        AppLocalizations.of(context).canNotUploadFileToSignature
+      );
+      return;
+    }
+
+    listImages
+      .map((fileInfo) => fileInfo.toPlatformFile())
+      .forEach((platformFile) => _insertInlineImage(
```

Comment:
Please use `for` or `Future.forEach`  to handle async

Reply:
Done